.github/workflows/ci.yml

```diff
@@ -90,3 +97,23 @@ jobs:
           SCALA_SCALA_DIR: scala_scala
           DOTTY_DIR: dotty
         run: script/smoke_test.sh
+
+      - name: copy nvim-treesitter highlight queries
+        if: ${{ runner.os == 'Linux' }}
+        shell: bash
+        run: cp ./nvim_treesitter/queries/scala/highlights.scm ./queries/highlights.scm
+
+      - name: Check if highlight queries are out of sync with nvim-treesitter
+        if: ${{ runner.os == 'Linux' }}
+        uses: tj-actions/verify-changed-files@v13
+        id: verify-changed-files
+        with:
+          files: |
+            queries/highlights.scm
```

Comment:
Actually, grammar (but not scanner, I think) changes can also trigger broken highlighting.

Reply:
Sure, but that will be caught by our own tests. The only reason we are looking here is to see if the highlights are different in our repo than that of the nvim-treesitter repo since this check is after we copied them.